scripts/tests_analyzer/pytest_marker_analyzer.py

```
@@ -883,37 +951,324 @@ def _extract_fixtures_from_file(file_path: Path, marker_names: set[str]) -> set[
+    """
+    changed_lines: set[int] = set()
+    current_line = 0
+
+    for line in diff_content.splitlines():
+        if line.startswith("@@"):
+            match = re.search(pattern=r"\+(\d+)", string=line)
+            if match:
+                current_line = int(match.group(1))
+            continue
+
+        if line.startswith("---") or line.startswith("+++"):
```

Ruff (0.15.2)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)